SocialFilter/sections/specific.txt

```diff
@@ -6886,7 +6886,7 @@ diariodenavarra.es,elpais.com.uy,ovaciondigital.com.uy,eltiempo.com,heraldo.es,t
 stuttgarter-zeitung.de##.m-shares
 apnews.com##.fbShare.socialIcons
 apnews.com##.tweet.socialIcons
-vk.sportsbull.jp,yutura.net,mainichi.jp,ytranking.net,ucuzabilet.com,programiz.com,viasatsport.se##.social-area
+nanoha-vivid.tv,vk.sportsbull.jp,yutura.net,mainichi.jp,ytranking.net,ucuzabilet.com,programiz.com,viasatsport.se##.social-area
```

Comment:
I guess you haven't seen my [message](https://github.com/AdguardTeam/AdguardFilters/issues/132639) that website has low visitors.
We don't add rules in that case.

Reply:
sorry